pkg/framework/test_context.go

```diff
@@ -69,6 +81,7 @@ func RegisterFlags() {
 	flag.StringVar(&TestContext.ReportPrefix, "report-prefix", "", "Optional prefix for JUnit XML reports. Default is empty, which doesn't prepend anything to the default name.")
 	flag.StringVar(&TestContext.ReportDir, "report-dir", "", "Path to the directory where the JUnit XML reports should be saved. Default is empty, which doesn't generate these reports.")
 	flag.StringVar(&TestContext.ImageServiceAddr, "image-endpoint", "", "Image service socket for client to connect.")
+	flag.StringVar(&TestContext.TestImagesFilePath, "test-images-file", "", "Optional path to a YAML file containing references to custom container images to be used in tests.")
```

Comment:
could you also update the docs for this new flag?

Reply:
Good catch, added [here](https://github.com/kubernetes-sigs/cri-tools/pull/855/files#diff-10e708eb89e00450f0a16b04affd53bd08b1ac65c41b40b1c4e9d1dd7d10116dR55), thank you!